.github/ISSUE_TEMPLATE/failed_check.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Gradle Check Failure.
+labels: >test-failure bug
+---
+
+A gradle check workflow has failed after merge.
+
+PR: {{ env.workflow_url }}
+CommitId: {{ env.pr_from_sha }}
```

Comment:
Just an idea, should we tag the author here (if possible) so he/she gets notified about post merge failure?

Reply:
Added both the pusher and PR author as assignees.